sdk/translation/azure-ai-translation-text/azure/ai/translation/text/__init__.py

```diff
@@ -6,19 +6,21 @@
 # Changes may cause incorrect behavior and will be lost if the code is regenerated.
 # --------------------------------------------------------------------------
 
-from ._patch import TextTranslationClient
+from ._client import TextTranslationClient
```

Comment:
Based on this diff, it looks like we've run the generation post-processing script previously, but didn't this time when we regenerated. The post-processing script will import anything you've patched from _patch in the init, instead of _client so that mypy can recognize the changes you made (like adding overloads). Let's regenerate with the post-processing script and see if it fixes our issue.

Add 

```toml
[tool.generate]
autorest-post-process = true
```

to your [pyproject.toml](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/translation/azure-ai-translation-text/pyproject.toml) file and then run tox to regenerate. By adding to the pyroject.toml this ensures that the postprocessing script will run every time after generation.

Reply:
Added and  have executed the TypeSpec-Project-Generate.ps1 script which is calling

```
npx tsp compile C:\src\azure-sdk-for-python\sdk\translation\azure-ai-translation-text\TempTypeSpecFiles\Azure.AI.TextTranslation\main.tsp --emit @azure-tools/typespec-python --option @azure-tools/typespec-python.emitter-output-dir=C:\src\azure-sdk-for-python\sdk\translation\azure-ai-translation-text/
```

But I can see that the issue is still present: https://dev.azure.com/azure-sdk/public/_build/results?buildId=3544462&view=logs&j=b70e5e73-bbb6-5567-0939-8415943fadb9

Reply:
Fixes are in, working as expected.